src/display/text_layer.js

```diff
@@ -27,8 +27,7 @@ import {
  *   render (the object is returned by the page's `getTextContent` method).
  * @property {ReadableStream} [textContentStream] - Text content stream to
  *   render (the stream is returned by the page's `streamTextContent` method).
- * @property {DocumentFragment} container - The DOM node that will contain the
- *   text runs.
+ * @property {Node} container - The DOM node that will contain the text runs.
```

Comment:
How about explicitly listing `{DocumentFragment | HTMLElement}` instead, since that's possibly slightly clearer here?

Reply:
I think that makes sense. Updated.